### ISSUE TITLE
ARM 32bits build

### ISSUE THREAD

**Issue by petrvecera** [jlesage/docker-jdownloader-2#131]
_First - thank you very much for the docker file! It's pretty cool! :)_ 

Would it be possible to build also arm/32bits version? 

Because most of the RaspberryPI OS are running on 32b (it has wider support than 64b) 

Thank you
🙏

**Comment by jlesage** [comment]
The problem is that, to my knowledge, there is no 32-bit ARM Java support for Alpine Linux.

> it has wider support than 64b

You are talking about the hardware or software ?  I agree that the 32bit OS is running on all Raspberry hardware, which is not the case for the 64bit version.  But on the software side, It seems that the 64bit version has more/better support.

**Comment by petrvecera** [comment]
Yeah 64bit are better. But I tried 64bit recently on rasp and had major issues with missing packages and libraries which were not yet created for 64bit. 

I managed to build it locally on 32bit ARM. But it was stuck in the updater loop, first updater run for ~5 minutes, than verification. And that got stuck. After restart updated was stuck in infinite loop.

**Comment by jlesage** [comment]
Java 8 for ARM32 in Alpine repository is running in interpreter mode only, meaning that it is slow as hell and not really usable...

**Comment by AlexeiScherbakov** [comment]
1. Don't use docker if you have less 4G memory on Raspberry PI
2. Use 64-bit mode if you have 4G or more on Arm

If you have less than 4G you must install JDownloader as regual app. Some years ago I use this script for installing Java on 32-bit ARM. It was BellSoft Java 12 
```
sudo apt-get install dirmgr
	sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv-keys 32E9750179FCEA62
	wget https://download.bell-sw.com/java/12.0.1/bellsoft-jdk12.0.1-linux-arm32-vfp-hflt.deb
	sudo apt-get install ./bellsoft-jdk12.0.1-linux-arm32-vfp-hflt.deb
	sudo update-alternatives --config javac
	sudo update-alternatives --config java
```
